src-ts/tools/learn/assets/data/courses.json

```diff
@@ -11,6 +11,11 @@
                 "First, you'll build a cat photo app to learn the basics of HTML and CSS. Later, you'll learn modern techniques like CSS variables by building a penguin, and best practices for accessibility by building a web form.",
                 "Finally, you'll learn how to make webpages that respond to different screen sizes by building a Twitter card with Flexbox, and a complex blog layout with CSS Grid."
             ],
+            "keyPoints": [
```

Comment:
These `keyPoints` are a duplicate of the `introCopy` -- this text should be different, right?

Reply:
Hmm, I think they should be different in the end, but this is what I see in the "MVP Copy for UI" document: https://topcoder.atlassian.net/wiki/spaces/PROD/pages/2916548609/MVP+Copy+for+UI#Learn-Detail-Page---New-user-or-in-progress-learning